src/gui/editors/TrackContainerView.cpp

```diff
@@ -349,6 +356,28 @@ void TrackContainerView::setPixelsPerBar( int ppb )
 	}
 }
 
+void TrackContainerView::setVerticalScale(double h)
+{
+	for (const auto& trackView : m_trackViews)
+	{
+		int new_height = lround(h * DEFAULT_TRACK_HEIGHT);
+		int cur_height = trackView->getTrack()->getHeight();
+
+		// If a track has been manually resized, calculate the new height by scaling the old height rather than just setting it.
+		if(cur_height != lround((m_trackHeightScale * DEFAULT_TRACK_HEIGHT)))
+			new_height = lround(cur_height * (h / m_trackHeightScale));
```

Comment:
```suggestion
		if (cur_height != lround((m_trackHeightScale * DEFAULT_TRACK_HEIGHT)))
		{
			new_height = lround(cur_height * (h / m_trackHeightScale));
		}
```

Reply:
Could also use \<cmath\>'s `std::lround` instead of the C-style `lround`

Reply:
since we're under `using namespace std` they should be equivalent, no?

Reply:
It looks like this file is one of the [few places](https://github.com/search?q=repo%3ALMMS%2Flmms+%22using+namespace+std%3B%22&type=code) in the codebase where we do that. I guess it's fine if you keep it, but I rather we didn't use `using namespace std` at all